dev/src/watch.ts

```diff
@@ -214,10 +233,11 @@ abstract class Watch<T = firestore.DocumentData> {
    * @internal
    *
    * @param firestore The Firestore Database client.
+   * @param _converter
```

Comment:
nit: Remove this `@param _converter` if you can, since it's unrelated to this PR.